src/ui/card/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {ReactElement} from 'react'
+import { Card as MUICard, Paper } from '@mui/material';
+import './_card.scss'
+
+enum TYPE  {
+
```

Comment:
During out conversation I mentioned that implement transparent background as shown in this image

![transparent](blob:https://antra-react-va.atlassian.net/b215f57c-5d42-4fdc-a730-922e0950722c#media-blob-url=true&id=7d69e2a9-0751-49d3-8abf-73ee53c58105&collection=&contextId=10017&mimeType=image%2Fpng&name=image-20220422-160525.png&size=238429&height=684&width=560)